dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/ConfigurationKey.java

```diff
@@ -60,12 +60,6 @@ public enum ConfigurationKey {
   /** SQL view write enabled, can be 'on', 'off'. (default: off) */
   SYSTEM_SQL_VIEW_WRITE_ENABLED("system.sql_view_write_enabled", Constants.OFF, false),
 
-  /**
-   * Disable server-side program rule execution, can be 'on', 'off'. <br>
-   * (default: on)
-   */
-  SYSTEM_PROGRAM_RULE_SERVER_EXECUTION("system.program_rule.server_execution", Constants.ON, false),
```

Comment:
Does this need to be mentioned in the release notes 🤷🏻

Reply:
It should have been mentioned in 2.41 release notes.
I also updated the docs, I will check with @Philip-Larsen-Donnelly or @ameenhere what is the best way to add this information somewhere